[thomas@tmb linux-2.6.27.18.test]$ LC_ALL=C make drivers/pci/
CHK     include/linux/version.h
CHK     include/linux/utsrelease.h
CALL    scripts/checksyscalls.sh
CC      drivers/pci/intel-iommu.o
drivers/pci/intel-iommu.c: In function 'iommu_flush_write_buffer':
drivers/pci/intel-iommu.c:530: error: 'rwbf_quirk' undeclared (first use in this function)
drivers/pci/intel-iommu.c:530: error: (Each undeclared identifier is reported only once
drivers/pci/intel-iommu.c:530: error: for each function it appears in.)
make[1]: *** [drivers/pci/intel-iommu.o] Error 1
make: *** [drivers/pci/] Error 2

Signed-off-by: Thomas Backlund <tmb@mandriva.org>

--- linux-2.6.27.18.test/drivers/pci/intel-iommu.c.orig	2009-02-19 00:50:49.000000000 +0200
+++ linux-2.6.27.18.test/drivers/pci/intel-iommu.c	2009-02-19 00:59:43.000000000 +0200
@@ -72,6 +72,8 @@ static struct deferred_flush_tables *def
 /* bitmap for indexing intel_iommus */
 static int g_num_of_iommus;
 
+static int rwbf_quirk = 0;
+
 static DEFINE_SPINLOCK(async_umap_flush_lock);
 static LIST_HEAD(unmaps_to_do);
 
@@ -1410,8 +1412,6 @@ static void domain_remove_dev_info(struc
 	spin_unlock_irqrestore(&device_domain_lock, flags);
 }
 
-static int rwbf_quirk = 0;
-
 /*
  * find_domain
  * Note: we use struct pci_dev->dev.archdata.iommu stores the info
